public/create_marked_entity.php

```diff
@@ -10,11 +10,10 @@
 $marked_entity->description = $_POST["description"];
 $marked_entity->is_group_work = $_POST["is_group_work"];
 // TODO: Fix this when courses are fully implemented
-$marked_entity->course_offering_id = $_POST["course_offering_id"] ?? 1;
+$marked_entity->course_offering_id = intval($_POST["course_offering_id"] ?? 1);
```

Comment:
course offering Id should be a select dropdown, not a text input, so it is sure that the course offering id is a valid one